dracut/30ignition/coreos-remove-initramfs-network.service

```diff
@@ -0,0 +1,17 @@
+# Clean up the initramfs networking on first boot
+# so the real network is being brought up
+
+[Unit]
+Description=Tear down initramfs networking
+DefaultDependencies=false
+After=ignition-files.service
+Before=initrd.target
+
+# Make sure if ExecStart= fails, the boot fails
+OnFailure=emergency.target
+OnFailureJobMode=isolate
+
+[Service]
+Type=oneshot
+ExecStart=/usr/sbin/coreos-remove-initramfs-network
+RemainAfterExit=yes
```

Comment:
So one thing that would be cool here is to follow the pattern `coreos-mount-var.service` & `ignition-complete.target` use, e.g.:

```
[Unit]
...

# Make sure ExecStop= runs before we switch root
Conflicts=initrd-switch-root.target umount.target
Before=initrd-switch-root.target

# Make sure if ExecStop= fails, the boot fails
OnFailure=emergency.target
OnFailureJobMode=isolate

[Service]
Type=oneshot
RemainAfterExit=yes
ExecStop=...
```

That way we tear down the network just before switching root.